content/en/references/localstack-cli-manual.md

```diff
@@ -0,0 +1,373 @@
+---
+title: LocalStack CLI manual
+weight: 3
+description: The reference guide for LocalStack CLI commands and how to get started on using them!
```

Comment:
ATM, it does not really show "how to get started on using them" (beyond the single completion example).